BiasAndFairnessModule/run_full_evaluation.py

```
@@ -0,0 +1,345 @@
+            race_value = convert_metric_to_float(race_result, metric_name)
+            all_results[f"{metric_name}_race"] = race_value
+            
+            print(f"   ✅ {metric_name}: sex={sex_value:.4f}, race={race_value:.4f}")
+            
+        except Exception as e:
+            print(f"   ❌ {metric_name}: Error - {str(e)}")
+            all_results[f"{metric_name}_sex"] = None
+            all_results[f"{metric_name}_race"] = None
+    
+    # 2. SECOND PRIORITY: Additional Fairness Compass recommended metrics
+    print(f"\n🧭 2. Fairness Compass Additional Metrics:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)